kadi/commands/commands_v2.py

```diff
@@ -353,43 +359,61 @@ def update_archive_and_get_cmds_recent(
             cmd_event["Date"], cmd_event["Event"], cmd_event["Params"]
         )
 
-        # Events that do not generate commands (e.g. load interrupt) return
-        # None from get_cmds_from_event.
+        # Events that do not generate commands return None from
+        # get_cmds_from_event.
         if cmds is not None and len(cmds) > 0:
             cmds_list.append(cmds)
-            rltts.append(None)
 
     # Sort cmds_list and rltts by the date of the first cmd in each cmds Table
     cmds_starts = np.array([cmds["date"][0] for cmds in cmds_list])
     idx_sort = np.argsort(cmds_starts)
     cmds_list = [cmds_list[ii] for ii in idx_sort]
-    rltts = [rltts[ii] for ii in idx_sort]
 
-    for ii, cmds, rltt in zip(itertools.count(), cmds_list, rltts):
-        if rltt is not None:
-            # Apply RLTT from this load to the current running loads (cmds_all).
-            # Remove commands with date greater than the RLTT date. In most
+    # Apply RLTT and any END SCS commands (CODISAXS) to loads. RLTT is applied
+    # by stopping SCS's 128-133.
+    for ii, cmds in enumerate(cmds_list):
+        cmds_source = cmds["source"][0]
+        if cmds_source == "CMD_EVT":
+            cmds_source = f"CMD_EVT {cmds[0]['event']} at {cmds[0]['date']}"
+        logger.info(f"Processing {cmds_source} with {len(cmds)} commands")
+        end_scs = collections.defaultdict(list)
+        if date_end := cmds.meta.get("rltt"):
+            source = f'RLTT in {cmds["source"][0]}'
+            end_scs[date_end, source].extend([128, 129, 130, 131, 132, 133])
```

Comment:
Makes sense to me to interpret RLTT as a "stop" for those SCSs, but is that the definition?  I've forgotten and don't know where to find that definition.

Reply:
Yes that's the definition. Running Loads Termination Time, where running loads are the weekly loads in SCSs 128-133 inclusive.